deployment.yaml

```diff
@@ -16,5 +16,5 @@ spec:
     spec:
       containers:
       - name: logger
-        image: ctovena/logger:0.1
+        image: logging/logger:0.1
```

Comment:
Are we going to host this image here https://quay.io/organization/openshift-logging? If yes, then image should be: `quay.io/openshift-logging/cluster-logging-load-client:0.1`

Reply:
moved to `quay.io/openshift-logging/cluster-logging-load-client:0.1`
DONE